overview.tt

```diff
@@ -0,0 +1,279 @@
+[% WRAPPER layout.tt title="Overview" handlesLayout=1 %]
+
+<div class="page-title">
+  <h1>Overview</h1>
+</div>
+
+<section class="overview-description">
+  Step on a journey to explore and learn why so many developers, system
+  administrators and other enthusiasts use Nix and can not imagine the world
+  without it.
+</section>
+
+<section class="overview-benefits">
+
+  <div>
+    <!-- sparkles, success, joy, for developers -->
+    [% PROCESS svg path="site-styles/assets/gfx-overview-cloud.svg" %]
+    <div>
+      <h2>Always working development environment.</h2>
+      <!--
+      <h2>Never breaking development environment.</h2>
+      <h2>Complete development environment.</h2>
+      <h2>Complete and consistent development environment.</h2>
+      -->
+      <p>
+        With Nix every developer will enter a
+        <strong>complete and consistent development environment</strong>.
+        Stop worrying how to install dependencies for your project.
+      </p>
+      <ul>
+        <!-- TODO: this should be links to a more detailed explanation
+                   in which we should explain how it works? and why it works
+                   the way it is works?
+        -->
+        <li>complete dependencies</li>
+        <li>build sandbox</li>
+      </ul>
+      <div>
+        <p>
+          Under Nix, a build process will only find resources that have been
+          declared explicitly as dependencies. There is no way it can build
+          until everything it needs has been correctly declared. If it builds,
+          you will know you've provided a complete declaration.
+        </p>
+        <p>
+          The same build process specification is than used to provide
+          development environment. With successful build, we can guarantee that
+          everything needed to build the project is available. Even more,
+          additional development tools can be added to assist the development.
+        </p>
+        <p>
+          Long are gone the days developers saying:
+          <strong>But it works on my machine!</strong>
+        </p>
+      </div>
+    </div>
+  </div>
+
+  <div>
+    <!-- collaboration, sharing -->
+    [% PROCESS svg path="site-styles/assets/gfx-overview-gears.svg" %]
+    <div>
+      <h2>
+        <!-- TODO: missing the emotional part
+        -->
+        One tool, many languages.
+      </h2>
+      <p>
+        With today's polyglot development environments, each language has its
+        own way of building and testing. Encourage cross-team development by
+        providing a single way of building everything.
+      </p>
+      <ul>
+        <li>language agnostic</li>
+        <li>portable</li>
+      </ul>
+      <div>
+        <p>
+          Keeping the project's installation instructions up to date is a never
+          ending task. Nix allows you to specify your system dependencies
+          alongside your project's dependencies in a declarative way.
```

Comment:
Not very clear here what the distinction between "system dependencies" and "project dependencies" is (especially since with Nix they're *all* project dependencies). Probably this should say something about dependencies that can be handled by language-specific package managers vs dependencies that cannot. Don't know a short way to phrase that though.